src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
Oh right, yes, we need sizes to skip.

Ok, how about just replacing the name "WLL" with "other" or something more generic. That seems fine for now.